patches/minecraft/net/minecraft/entity/boss/EntityDragon.java.patch

```diff
@@ -443,7 +443,18 @@
          }
  
          boolean flag = this.field_70170_p.func_82736_K().func_82766_b("doMobLoot");
-@@ -750,7 +642,7 @@
+@@ -741,7 +633,9 @@
+         {
+             if (this.field_70995_bG > 150 && this.field_70995_bG % 5 == 0 && flag)
+             {
+-                this.func_184668_a(MathHelper.func_76141_d((float)i * 0.08F));
++                int b = MathHelper.func_76141_d((float)i * 0.08F);
++                b = net.minecraftforge.event.ForgeEventFactory.getExperienceDrop(this, this.field_70717_bb, b);
++                if(b>0) this.func_184668_a(b);
```

Comment:
Instead of inserting the `getExperienceDrop` here, do it in this method instead:

```java
private void dropExperience(int p_184668_1_)
{
    // Insert it here instead
    while (p_184668_1_ > 0)
    {
         int i = EntityXPOrb.getXPSplit(p_184668_1_);
         p_184668_1_ -= i;
         this.world.spawnEntity(new EntityXPOrb(this.world, this.posX, this.posY, this.posZ, i));
    }
}
```